src/api/providers/openrouter.ts

```diff
@@ -278,7 +278,7 @@ export async function getOpenRouterModels() {
 					modelInfo.supportsPromptCache = true
 					modelInfo.cacheWritesPrice = 3.75
 					modelInfo.cacheReadsPrice = 0.3
-					modelInfo.maxTokens = 64_000
+					modelInfo.maxTokens = rawModel.id === "anthropic/claude-3.7-sonnet:thinking" ? 64_000 : 16_384
```

Comment:
It seems like max tokens should be 64k even for non-thinking - or am I not understanding this right?

Reply:
My understanding is that we should only set it to 64K if we allow you to customize the `maxTokens` that we pass in the completion API calls, and this is only the case for the thinking models (as of now).